src/ApolloHooksMutation.re

```diff
@@ -64,6 +64,8 @@ type options('a) = {
   [@bs.optional]
   optimisticResponse: Js.Json.t,
   [@bs.optional]
+  errorPolicy: string,
```

Comment:
looks good, probably just leave a link for defining it as a poly. variant in some next PR?
https://www.apollographql.com/docs/react/data/error-handling/#error-policies

Reply:
Opened #132.